### ISSUE TITLE
Ability to set dynamic values for timeout in HttpSender

### ISSUE THREAD

**Issue by alisihab** [frankframework/frankframework#5239]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
Currently, I am building a dispatcher that will read configuration files and forward the incoming requests to the appropriate service.
While sending the requests to the other systems, I need to read the timeout  setting from the config file then configure it in HttpSender. Therefore it would be nice to be able to configure the timeout information in HttpSender during runtime.

Moreover, the timeout setting should be dynamic for JmsSender as well.

**Reporter**
Ali

**Comment by nielsm5** [comment]
@alisihab is this still needed? Because the solution will complicate the FF code considerably... ☹

**Comment by alisihab** [comment]
Our current solution involves the use of multiple httpSenders and jmsSenders, each configured with a predefined timeout. This approach has resulted in too many senders, each differing only by their timeout values. Although this is not ideal, it has been manageable because the timeout values were predetermined.

However, in scenarios where the timeout values are not known in advance, it is necessary to implement a mechanism that allows for dynamic timeout configuration in the senders.

**Comment by nielsm5** [comment]
I personally just don't understand when a timeout can be dynamic.. A process should last max X seconds, and the timeouts should be set accordingly. If the timeout is dynamic, what's the point of it?

**Comment by alisihab** [comment]
Indeed, that is correct. However, in this case, processes A, B, and C utilize the same httpSender/jmsSender to send messages to different systems, each with distinct timeout values. As mentioned in the issue description, this functions as a dispatcher.

It is not that process A alternates between X and Y seconds timeouts
